.github/workflows/build.yml

```diff
@@ -0,0 +1,30 @@
+name: build
+
+on: [push]
+
+jobs:
+  tests:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 120
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y astyle libcunit1-dev libaio-dev libssl-dev \
+           libnuma-dev uuid-dev lcov nasm meson ninja-build python3-pyelftools
+          git submodule update --init
```

Comment:
Instead of checking out submodule separately, can use the `actions/checkout` like [this](https://github.com/actions/checkout#usage)